pkg/controller/instance/instance_controller_test.go

```diff
@@ -128,6 +128,13 @@ func TestInstanceController_Transition_Installing(t *testing.T) {
 
 	r := buildReconcileWithFakeClientWithMocks([]runtime.Object{&i, &instanceSecret})
 
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mockClient := helmclientmock.NewMockClient(ctrl)
+	r.helmClientReceiver = func(repoCache, repoConfig, namespace string) (helmclient.Client, error) {
```

Comment:
You invoke a mocked version of the helm client without ensuring it's methods are called.